.env.development

```diff
@@ -7,7 +7,11 @@
 DATABASE_URL=postgresql://postgres@localhost/laa-review-criminal-legal-aid
 
 DEVELOPMENT_HOST=laa-review-criminal-legal-aid.test
-CRIME_APPLY_API_URL=https://laa-apply-for-criminal-legal-aid.test/api/
```

Comment:
I've removed this env as it seems is no longer used anywhere? Please let me know otherwise and I will revert the change!

Reply:
👍